src/main/resources/META-INF/plugin.xml

```diff
@@ -181,19 +181,29 @@
 
     <actions>
 
-        <group id="KorgeTest" >
+        <group id="KorgeTest">
             <add-to-group group-id="NavBarToolBar" anchor="last"/>
-            <separator />
+            <separator/>
             <!--
             <reference ref="KorgeAccountAction" />
             <action id="KorgeKorgeAction" class="com.soywiz.korge.intellij.actions.KorgeKorgeAction" text="_Korge"
                     icon="/com/soywiz/korge/intellij/icon/korge.png"
             />
             -->
-            <action id="KorgeAccountAction" class="com.soywiz.korge.intellij.actions.KorgeAccountAction" text="_Account"
-                    icon="AllIcons.General.User"/>
-            <action id="KorgeStoreAction" class="com.soywiz.korge.intellij.actions.KorgeStoreAction" text="KorGE _Store"
-                    icon="AllIcons.Actions.Install"/>
+            <action id="KorgeAccountAction"
+                class="com.soywiz.korge.intellij.actions.KorgeAccountAction" text="_Account"
+                icon="AllIcons.General.User"/>
+            <action
+                id="KorgeStoreAction"
+                class="com.soywiz.korge.intellij.actions.KorgeStoreAction"
+                text="KorGE _Store"
+                icon="AllIcons.Actions.Install"/>
+            <action
+                id="KorgeFixGradleMavenLocalSourcesAction"
+                class="com.soywiz.korge.intellij.actions.KorgeFixGradleMavenLocalSourcesAction"
+                text="Fix Maven Local Sources"
+                description="Try clicking this if you're not seeing sources for maven local build of Korge. "
+                icon="AllIcons.Actions.BuildLoadChanges"/>
```

Comment:
I don't think we should add a button here. Why is this required? Isn't this working properly after synchronizing? In which cases is this needed?

Reply:
Could be useful in the sense that it makes people aware of the feature, and if it doesn't work as they expected, then they will make a feature request.

Also, people can still hide the button by right clicking and customizing the toolbar.

Reply:
I see. But I think most features users shouldn't be directly aware in the UI. Maybe we can put that the functionality is there in the plugin description, but I believe we should save space or otherwise it would end polluted. For example I decided to put the documentation in the login section to reduce one button from the toolbar. In fact recent IntelliJ versions removed buttons instead of adding new ones. We can still allow the users to add them later.